Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.yml

```
@@ -8,18 +11,24 @@ configuration:
   name: url
   required: true
   type: 0
+  section: Connect
 - display: Username
   name: credentials
   required: true
   type: 9
+  section: Connect
 - display: Trust any certificate (not secure)
   name: insecure
   type: 8
   required: false
+  section: Connect
+  advanced: true
 - display: Use system proxy settings
   name: proxy
   type: 8
   required: false
+  section: Connect
+  advanced: true
 description: ANY.RUN is a cloud-based sanbox with interactive access.
```

```suggestion
description: ANY.RUN is a cloud-based sandbox with interactive access.
```
As per Richard's suggestion